flightros/include/flightros/record.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
```

Comment:
the `record.hpp` can be more specific, e.g., "record_events.hpp"

Reply:
but it is actually to record all kind of data, e.g. positions, images, ...